### PR TITLE
calculate bottom margin based on width/11

### PR DESCRIPTION
**Pull request by hsab**
This is a minor fix for the bottom margin. But on Large screens the 95px hardcoded value does not suffice and there's still some overlapping. 

I made sure the bottom of the page and the nav menu always line up.
If page is in landscape mode (blobs are in a column on the side), then we assign a hardcoded 7vh to both `margin-bottom`s.
If in portrait (blobs at the bottom in a row) this value is `calc(100vw/11)` which is the exact formula used for blobs diameter, so alignment is perfect.

Purely css so no js mixups. 

Screenshot_20200525_121957
Screenshot_20200525_122007
Screenshot_20200525-122038

### CODE REVIEW

File: src/Components/Nav/style.module.scss [hunk 1]
```diff
@@ -16,6 +16,10 @@
     margin-left: var(--page-gutter);
     margin-bottom: 7vh;
   }
+  @include portrait {
+    margin-left: var(--page-gutter);
```
Comment (hsab) [inline src/Components/Nav/style.module.scss]: this is unnecessary. copa pasta

File: src/Containers/Artist/style.module.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import "../../common/util.scss";
+
+
+.container {
+  margin-bottom: 7vh;
+  @include portrait {
+    margin-left: var(--page-gutter);
```
Comment (blerchin) [inline src/Containers/Artist/style.module.scss]: This makes the whole page uncentered?

**Comment by blerchin** [comment]
Calculating the margin-bottom makes sense to me. But I'm confused why either of the `margin-left`s are there

File: src/Containers/Artist/style.module.scss (hunk 2 above)
Comment (hsab) [reply src/Containers/Artist/style.module.scss]: thanks for catching that. yeah that's copy pasta from
```
  @include medium {
    margin-left: var(--page-gutter);
    margin-bottom: 95px;
  }
```

**Comment by hsab** [comment]
should be fixed now

**Review by blerchin — APPROVED** [review]

**Pull request merged**